pyfstat/make_sfts.py

```diff
@@ -938,6 +797,112 @@ def __init__(
             np.array([delta_phi, delta_F0, delta_F1, delta_F2]).T
         )
 
+    def get_base_template(self, i, Alpha, Delta, h0, cosi, psi, phi, F0, F1, F2, tref):
```

Comment:
```suggestion
    def get_base_template(self, i, Alpha, Delta, h0, cosi, psi, phi, F0, F1, F2, tref):
        """ FIXME: ported over from Writer, should be replaced by a more elegant re-use of parse_args_consistent_with_mfd """
```
(black may require linebreaks or something)